packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -238,244 +261,139 @@
+		},
+		"25[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable_inverted",
+			"label": "Send Command Frames in 4th and 5th Association Groups when Switching ON (Single Click)",
+			"defaultValue": 0
+		},
+		"25[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable_inverted",
+			"label": "Send Command Frames in 4th and 5th Association Groups when Switching OFF (Single Click)",
+			"defaultValue": 0
+		},
+		"25[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable_inverted",
+			"label": "Send Command Frames in 4th and 5th Association Groups when Changing Dimming Level",
+			"defaultValue": 0
+		},
+		"25[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable_inverted",
+			"label": "Send Command Frames in 4th and 5th Association Groups on Double Click",
+			"defaultValue": 0
+		},
+		"25[0x10]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Send Command Frames in 4th and 5th Association Groups with 0xFF Value on Double Click"
 		},
 		"26": {
-			"label": "The function of 3-way switch",
-			"description": "Key no. 2 controls the Dimmer additionally",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "3-way switch function for S2 disabled",
-					"value": 0
-				},
-				{
-					"label": "3-way switch function for S2 enabled",
-					"value": 1
-				}
-			]
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "3-way Switch Function",
+			"description": "Key S2 also controls the dimmer"
 		},
-		"27": {
-			"label": "Associations in Z-Wave network security mode",
-			"description": "Defines commands are sent as secure or non-secure",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 15,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "all groups (II-V) sent as non-secure",
-					"value": 0
-				},
-				{
-					"label": "2nd group sent as secure",
-					"value": 1
-				},
-				{
-					"label": "3rd group sent as secure",
-					"value": 2
-				},
-				{
-					"label": "4th group sent as secure",
-					"value": 4
-				},
-				{
-					"label": "5th group sent as secure",
-					"value": 8
-				},
-				{
-					"label": "all groups (II-V) sent as secure",
-					"value": 15
-				}
-			]
+		"27[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Send secure Z-wave commands to 2nd association group"
+		},
+		"27[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Send secure Z-wave commands to 3rd association group"
+		},
+		"27[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Send secure Z-wave commands to 4th association group"
```

Comment:
Capitilization is off. The labels should be Title Case. Please carry throughout.

Reply:
Correct, these used to be option labels.